docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+
+  document-merge-service:
+    image: adfinissygroup/document-merge-service@sha256:bfcfb01877d59bb2a73c3f21d435e6abde6f2b556a3f9c335238ecf9f892cadc
```

Comment:
Why are you referring to a specific sha here?

Reply:
We don't have versions for the document-merge-service.

But, as this is for development only, we could use `:latest` and have the developer debug possible problems with new versions.

I'm open for both approaches :smiley:

Reply:
I personally think `:latest` would probably be the better approach depending when the image gets built. If its on every master merge, you would have to update the `docker-compose.yml` file on each pr merged in the document-merge-service repo. But i will leave this up to you to decide :+1:

Reply:
Et voilà :sunglasses: